Source/DfBAdminToolkit.Common/Services/MemberServices.cs

```diff
@@ -145,6 +145,10 @@ public IServiceResponse AddMember(IMemberData data, string authToken)
                         ),
                         new JProperty("force_async", false)
                     );
+                    if (data.PersistentId != null) {
```

Comment:
We should probably change this to `!String.IsNullOrEmpty(data.PersistentId)` if the data table columns are editable.
```suggestion
                    if (!String.IsNullOrEmpty(data.PersistentId)) {
```